backend/src/game.h

```diff
@@ -46,7 +47,7 @@ class Game {
                 ((MiniGame*)current_gamestate)->start();
 
                 // fixme: Infinite loop for now
-                miniGames.push(new CrazyCounting_MiniGame(this));
```

Comment:
Push both minigames into the queue when miniGames.Length is below 2 or something